articles/deploy-fleet-on-cloudgov.md

```diff
@@ -110,7 +112,10 @@ variables](https://fleetdm.com/docs/deploying/configuration#using-only-environme
 }
 ```
 
-<meta name="title" value="Cloud.gov">
-<meta name="pageOrderInSection" value="700">
-<meta name="description" value="A guide for deploying Fleet on Cloud.gov.">
-<meta name="navSection" value="Deployment guides">
+<meta name="articleTitle" value="Deploy Fleet on Cloud.gov">
+<meta name="authorGitHubUsername" value="mike-j-thomas">
+<meta name="authorFullName" value="Mike Thomas">
```

Comment:
Should this be attributed to the GH user who added the original documentation page? (https://github.com/fleetdm/fleet/pull/7611)

```suggestion
<meta name="authorGitHubUsername" value="JJediny">
<meta name="authorFullName" value="John Jediny">
```